extensions/amp-selector/1.0/amp-selector.js

```diff
@@ -249,12 +250,42 @@ function OptionShim({
 function SelectorShim({
   shimDomElement,
   children,
+  form,
   multiple,
+  name,
   disabled,
   onKeyDown,
   role = 'listbox',
   tabIndex,
+  value,
 }) {
+  const input = useRef(
+    createElementWithAttributes(shimDomElement.ownerDocument, 'input', {
+      'hidden': '',
+    })
+  );
+
+  useLayoutEffect(() => {
+    const el = input.current;
+    shimDomElement.appendChild(el);
+    return () => shimDomElement.removeChild(el);
+  }, [shimDomElement, input]);
+
+  const syncAttr = useCallback(
+    (attr, value) => {
+      if (value) {
```

Comment:
Is an empty string a valid value here? At least with the "value" attribute, usually, an empty string is ok.

Reply:
Good point. I assume we don't want to propagate empty `form` and `name` attributes. Isn't `[value=""]` equivalent to `value` being omitted for input elements? If so this should be fine as is, however could be error prone if we were to tack on additional attributes this way.

Reply:
Yeah. I think it's pretty likely. If the case - then we can keep as is.